.github/pull_request_template.md

```diff
@@ -1,7 +1,35 @@
-This PR implements issue(s) #
+## Description
+<!-- Provide a brief description of the changes in this PR -->
 
-### Checklist
+## Related Issue
 
-* [ ] I wrote new tests for my new core changes.
-* [ ] I have successfully ran tests, style checker and build against my new changes locally.
-* [ ] I have informed the team of any breaking changes if there are any.
\ No newline at end of file
+This PR addresses/relates to issue(s) #
+<!-- If this PR addresses an existing issue, please link to it here -->
+<!-- e.g., Fixes #123 -->
+
+## Type of Change
+<!-- Mark the appropriate option with an "x" -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
```

Comment:
Do we need an additional section for `hotfix` or do you think `Bug fix` already covers it?